StripeCore/StripeCore/Source/API Bindings/STPAPIClient.swift

```diff
@@ -87,6 +87,10 @@ import UIKit
         return publishableKey?.hasPrefix("uk_") ?? false
     }
 
+    /// Determines the `Stripe-Livemode` header value when the publishable key is a user key
+    /// :nodoc:
```

Comment:
Do we still need `:nodoc:`? I remember that being necessary for spi-public stuff earlier to let Jazzy know to ignore it.

Reply:
nope, can remove, we use DocC now